corehq/apps/smsbillables/models.py

```diff
@@ -272,6 +273,14 @@ class Meta:
 
     @property
     def gateway_charge(self):
+        return self.multipart_count * self._single_gateway_charge
```

Comment:
@gcapalbo I think we can get around the issue in https://github.com/dimagi/commcare-hq/pull/10220#issuecomment-205765594 by modifying https://github.com/dimagi/commcare-hq/pull/10220 to branch here if ```direct_gateway_fee``` is not ```None```